butterfree/load/writers/historical_feature_store_writer.py

```diff
@@ -119,21 +119,9 @@ def write(
             feature_set: object processed with feature_set informations.
             dataframe: spark dataframe containing data from a feature set.
             spark_client: client for spark connections with external services.
-
-        If the debug_mode is set to True, a temporary table with a name in the format:
-        historical_feature_store__{feature_set.name} will be created instead of writing
-        to the real historical feature store.
-
         """
         dataframe = self._create_partitions(dataframe)
```

Comment:
Another points is that this columns need to be created on the df to ve verified by the check schema hook. So when instantiating the `HistoricalFeatureStore` the constructor need to add a "create_ymd_partitions" hook and sequence the spark schema hook.

Reply:
About the renaming of the method, I agree, but about the creation of pre_hooks for the dataframe, it may be something cool but it would not be part of this PR.
As for the partition check, the Spark check hook does not check this.

Reply:
I don't agree. In my opinion this is something required for this PR to work as it should. The main point:
- year/month/day columns need to be in the df before checking the schema against the spark table. The way to do so is by having a pre_hook before the check schema hook. It does not make sense to check the schema and after changing the schema and writing to the table.
- the point about the debug mode is not required but doing with a hook would not break the debug mode tests. The views created using the historical feature store writer in debug mode would have the year/month/day as it should.